third_party/2and3/dateutil/rrule.pyi

```diff
@@ -0,0 +1,103 @@
+from ._common import weekday as weekdaybase
```

Comment:
There's currently no stub for the `_common` submodule -- that's the reason for the main test failure. You can add it, moving the `weekday` class there from `relativedelta.pyi`. The flake8 failure looks like a temporary pypi outage, so let's ignore it for now. (Though flake8 3.5.0 is now available.)